core/metacling/src/TCling.cxx

```diff
@@ -2902,6 +2902,17 @@ Bool_t TCling::IsLoaded(const char* filename) const
    return kFALSE;
 }
 
+Bool_t TCling::IsLibLoaded(const char* libname) const
+{
+   R__LOCKGUARD(gInterpreterMutex);
+
+   // Check shared library.
+   if (fInterpreter->getDynamicLibraryManager()->isLibraryLoaded(libname))
```

Comment:
What about `return fInterpreter->getDynamicLibraryManager()->isLibraryLoaded(libname);`

Reply:
In the 'original' version (that is still in IsLoaded) the check was done **also/first** on 
```
gSystem->FindDynamicLibrary(sFilename, kTRUE);
```
can the commit message be expanded to explain why this is not (anymore) needed?